src/utils/combineReducers.js

```diff
@@ -1,7 +1,9 @@
+import { ActionTypes } from '../createStore';
+import invariant from 'invariant';
+import isPlainObject from '../utils/isPlainObject';
 import mapValues from '../utils/mapValues';
 import pick from '../utils/pick';
-import invariant from 'invariant';
-import { ActionTypes } from '../createStore';
+import warning from 'warning';
```

Comment:
Move invariant next to warning in the import order? Can do, I just put them into alphabetical order.